workflow/scripts/generate-overview-table.py

```diff
@@ -152,7 +153,12 @@ def register_contig_lengths(assemblies, name):
         #         varcount = f" ({varcount})"
         # pangolin_call = f"{lineage}{varcount}"
         pangolin_call = f"{lineage}"
-    data.loc[sample, "Lineage"] = pangolin_call
+    data.loc[sample, "Pangolin Lineage"] = pangolin_call
+    if scorpio == "None":
+        scorpio_call = "no clear name because of missing strain"
```

Comment:
```suggestion
        scorpio_call = "-"
```

This could maybe blow the width of the table, what do you think of the "-"?